plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPluginTest.kt

```diff
@@ -494,6 +494,42 @@ class KtlintPluginTest : AbstractPluginTest() {
         }
     }
 
+    @DisplayName("Lint check should run incrementally and repeat errors")
+    @CommonTest
+    fun checkIsIncrementalWithErrors(gradleVersion: GradleVersion) {
+        project(gradleVersion) {
+            val initialSourceFile = "src/main/kotlin/Initial.kt"
+            createSourceFile(
+                initialSourceFile,
+                """
+                val foo="bar"
+
+                """.trimIndent()
+            )
+
+            buildAndFail(CHECK_PARENT_TASK_NAME) {
+                assertThat(task(":$mainSourceSetCheckTaskName")?.outcome).isEqualTo(TaskOutcome.FAILED)
+            }
+
+            val additionalSourceFile = "src/main/kotlin/AnotherFile.kt"
+            createSourceFile(
+                additionalSourceFile,
+                """
+                val bar="foo"
+
+                """.trimIndent()
+            )
+
+            buildAndFail(CHECK_PARENT_TASK_NAME, "--info") {
+                assertThat(task(":$mainSourceSetCheckTaskName")?.outcome).isEqualTo(TaskOutcome.FAILED)
+                assertThat(output)
+                    .contains("Executing incrementally")
+                    .contains("Initial.kt")
```

Comment:
previously, Initial would not be listed (and if AnotherFile passed, the whole task would pass)